.storybook/main.ts

```diff
@@ -1,97 +1,31 @@
-import path from 'path';
-import { existsSync } from 'fs';
-import { readdir } from 'fs/promises';
-import type { StorybookConfig } from '@storybook/react-vite';
+import type { StorybookConfig } from '@storybook/react-webpack5';
 
 const config: StorybookConfig = {
-	framework: {
-		name: '@storybook/react-vite',
-		options: {},
-	},
-	core: {
-		builder: '@storybook/builder-vite',
-	},
-	features: {
-		storyStoreV7: false,
-		buildStoriesJson: true,
-	},
+	framework: '@storybook/nextjs',
```

Comment:
The actions tab in Storybook now prints out pre-fetched routes on hover but I think that's ok. I couldn't find a way to turn it off.